command/init.go

```diff
@@ -136,39 +269,63 @@ General Options:
-                            the master key.
+  -pgp-keys			If provided, must be a comma-separated list of
+				files on disk containing binary- or base64-format
+				public PGP keys, or Keybase usernames specified as
+				"keybase:<username>". The number of given entries
+				must match 'key-shares'. The output unseal keys will
+				be encrypted and hex-encoded, in order, with the
+				given public keys.  If you want to use them with the
+				'vault unseal' command, you will need to hex decode
+				and decrypt; this will be the plaintext unseal key.
 
-  -stored-shares=0          The number of unseal keys to store. This is not
-                            normally available.
+  -recovery-shares=5		The number of key shares to split the recovery key
+				into. This is not normally available.
 
-  -pgp-keys                 If provided, must be a comma-separated list of
-                            files on disk containing binary- or base64-format
-                            public PGP keys, or Keybase usernames specified as
-                            "keybase:<username>". The number of given entries
-                            must match 'key-shares'. The output unseal keys will
-                            be encrypted and hex-encoded, in order, with the
-                            given public keys.  If you want to use them with the
-                            'vault unseal' command, you will need to hex decode
-                            and decrypt; this will be the plaintext unseal key.
+  -recovery-threshold=3		The number of key shares required to reconstruct
+				the recovery key. This is not normally available.
 
-  -recovery-shares=5        The number of key shares to split the recovery key
-                            into. This is not normally available.
+  -recovery-pgp-keys		If provided, behaves like "pgp-keys" but for the
+				recovery key shares. This is not normally available.
 
-  -recovery-threshold=3     The number of key shares required to reconstruct
-                            the recovery key. This is not normally available.
+  -auto				If set, performs service discovery using the underlying
+				Consul storage backend. When one or more Vault servers
+				are running on Consul storage backend (none else),
+				setting this flag will create a Consul client and
+				discovrs the nodes using the service name under which
+				Vault nodes are registered with Consul. Service name
+				should be supplied using 'consul-service' flag. This
+				option works well when each Vault cluster is registered
+				under a unique service name. Ensure that environment
+				variables required to communicate with Consul, like
+				(CONSUL_HTTP_ADDR, CONSUL_HTTP_TOKEN, CONSUL_HTTP_SSL,
+				et al) are properly set. If, only one Vault node is
+				discovered, then an initialization attempt will be made.
+				If more than one Vault nodes are discovered, they will
+				be listed on the output, requiring another execution of
```

Comment:
I'd end it after "...be output", the output will tell them that they need to run it again.